src/platform/forms-system/src/js/validation.js

```diff
@@ -138,16 +143,16 @@ export function uiSchemaValidate(
   appStateData,
 ) {
   if (uiSchema && schema) {
-    const currentData = path !== '' ? _.get(path, formData) : formData;
+    const currentData = path !== '' ? get(path, formData) : formData;
```

Comment:
`path` might contain multiple properties, so everything in this function is fine :+1: